src/azureFunction/azureFunctionUtils.ts

```diff
@@ -149,11 +149,14 @@ export async function getAzureFunctionProject(): Promise<string | undefined> {
 export async function getAzureFunctionProjectFiles(): Promise<string[] | undefined> {
 	let projFiles: string[] = [];
 	const hostFiles = await getHostFiles();
-	for (let host of hostFiles) {
-		let projectFile = await vscode.workspace.findFiles('*.csproj', path.dirname(host));
-		projectFile.filter(file => path.dirname(file.fsPath) === path.dirname(host) ? projFiles.push(file?.fsPath) : projFiles);
+	if (hostFiles !== undefined) {
```

Comment:
This is honestly super minor, but alternatively we could do:

```
if (!hostFiles) {
    return undefined;
}
```

...and then leave the rest the same. Just would touch fewer lines and involve less scrolling.

Honestly, it's more of a style thing than anything. Feel completely empowered to ignore me, just throwing it out there 😄

Reply:
All good, this is a better solution. I changed it to this way originally just due to debugging.